legal-api/src/legal_api/resources/v2/business/colin_sync.py

```diff
@@ -76,6 +85,12 @@ def get_completed_filings_for_colin():
                 except Exception as ex:  # noqa: B902
                     current_app.logger.info(ex)
                     continue  # do not break this function because of one filing
+            elif (filing.filing_type == 'dissolution' and
+                  filing.filing_sub_type == 'involuntary'):
+                batch_processings = BatchProcessing.find_by(business_id=filing.business_id)
```

Comment:
I think there is a potential scenario where a business that has been involuntarily dissolved that gets restore and involuntarily dissolved again.  Your code would still take the first `batch_processing` entry which is ok as long as the sort order is correct.  But searching for the matching `batch_processing` entry using the `batch_processing.filing_id` is probably the safest.